<?xml version="1.0" encoding="UTF-8" ?>

<launch>
    <arg name="task" default="3" />
    <node pkg="localization" type="nuscenes_localization" name="nuscenes_localization" output="screen">
        <param name="task" value="$(arg task)" />        
        <param name="max_x" value="45" />
        <param name="min_x" value="5" />

        <param name="max_y" value="30" />
        <param name="min_y" value="5" />

        <param name="max_z" value="12" />
        <param name="min_z" value="3" />
    </node>

</launch>